plugins/woocommerce/tests/e2e-pw/tests/merchant/page-loads.spec.js

```diff
@@ -95,126 +100,85 @@ const wcPages = [
 				name: 'Coupons',
 				heading: 'Coupons',
 				element: '.page-title-action',
-				text: 'Add new coupon',
+				// WP6.6: "Add coupon", WP6.7: "Add new coupon"
+				text: /Add coupon|Add new coupon/,
 			},
 		],
 	},
 ];
 
-let productId, orderId;
-const productName = 'Simple Product Name';
-const productPrice = '15.99';
-
 for ( const currentPage of wcPages ) {
-	const randomNum = new Date().getTime().toString();
-	const customer = {
-		username: `customer${ randomNum }`,
-		password: 'password',
-		email: `customer${ randomNum }@woocommercecoree2etestsuite.com`,
-	};
 	test.describe(
 		`WooCommerce Page Load > Load ${ currentPage.name } sub pages`,
-		{ tag: [ tags.GUTENBERG, tags.SERVICES ] },
+		{ tag: [ '@gutenberg', '@services' ] },
```

Comment:
Is this an unintentional change? I see it was previously updated [here](https://github.com/woocommerce/woocommerce/pull/53580/files#diff-79afed9fd52d465bb5f30f5bdbe83ecf62ed5a8b49b88c6484eb43538da14b66R117) as a part of https://github.com/woocommerce/woocommerce/pull/53580

Reply:
Not intentional. There was a merge conflict with 53580 and it looks like I missed these when resolving.
Updated with https://github.com/woocommerce/woocommerce/pull/53624